idc/metadata_utils.py

```diff
@@ -48,7 +56,8 @@ def get_collex_metadata(filters, fields, with_docs=True):
         'fqs': query_set,
         'query_string': "*:*",
         'facets': solr_facets,
-        'limit': 10 if with_docs else 0,
+        'limit': record_limit if with_docs else 0,
+        # what is with_docs supposed to do?? 'limit': 10 if with_docs else 0,
         'collapse_on': 'case_barcode',
```

Comment:
'10' was just a temporary magic number; eventually an environment level setting was going in which would have replaced 10 (so settings.SOLR_FETCH_LIMIT). We can do it this way as well; it's six of one half dozen of the other, but there SHOULD be a setting in settings.py which is the maximum allowed.